merlin-driver/src/main/java/gov/nasa/jpl/aerie/merlin/driver/engine/SimulationEngine.java

```diff
@@ -138,6 +153,100 @@ private SimulationEngine(SimulationEngine other) {
     }
   }
 
+  /** Initialize the engine by tracking resources and kicking off daemon tasks. **/
+  public void init(Map<String, Resource<?>> resources, TaskFactory<Unit> daemons) throws Throwable {
+    // Begin tracking all resources.
+    for (final var entry : resources.entrySet()) {
+      final var name = entry.getKey();
+      final var resource = entry.getValue();
+
+      this.trackResource(name, resource, elapsedTime);
+    }
+
+    // Start daemon task(s) immediately, before anything else happens.
+    this.scheduleTask(Duration.ZERO, daemons);
+    {
+      final var batch = this.extractNextJobs(Duration.MAX_VALUE);
+      final var results = this.performJobs(batch.jobs(), cells, elapsedTime, Duration.MAX_VALUE);
+      for (final var commit : results.commits()) {
+        timeline.add(commit);
+      }
+      if(results.error.isPresent()) {
+        throw results.error.get();
+      }
+    }
+  }
+
+  public sealed interface Status {
+    record NoJobs() implements Status {}
+    record AtDuration() implements Status{}
+    record Nominal(
```

Comment:
Consider a different word @mattdailis

Reply:
Are NoJobs and AtDuration different? My understanding was that one implies the other

Reply:
If an activity is unfinished at the end of simulation, you will jobs but be AtDuration

Reply:
Having trouble coming up with a different word too - let's not block this PR on it